Geometry/CommonDetUnit/interface/GeomDetLess.h

```diff
@@ -0,0 +1,34 @@
+#ifndef TrackerGeometryBuilder_GeomDetLess_H
+#define TrackerGeometryBuilder_GeomDetLess_H
+
+#include "DataFormats/TrajectorySeed/interface/PropagationDirection.h"
```

Comment:
This include defines an enumerator. However, adding here a direct dependency on DataFormats/TrajectorySeed introduces circular dependency:

```
tmp/slc6_amd64_gcc530/src/Geometry/CommonDetUnit/src/GeometryCommonDetUnit/libGeometryCommonDetUnit.so <- tmp/slc6_amd64_gcc530/cache/prod/libGeometryCommonDetUnit
```